src/Server/src/main/java/com.parking/controller/SampleController.java

```diff
@@ -11,8 +14,13 @@
 @RestController
 public class SampleController {
 
+    @Autowired
+    private IQueueMessageSender sender;
+
     @RequestMapping(method = RequestMethod.GET)
     public ResponseEntity<ApiResponseBody> getSampleResponse() {
+         ImageMessage msg = new ImageMessage("abc", "test bytes".getBytes());
+         boolean succeeded = sender.send(msg);
         return new ResponseEntity<>(new ApiResponseBody("SampleResponse"), HttpStatus.OK);
```

Comment:
HttpStatus.PROCESSING and adjust response message.

Reply:
Well, I thought that you will implement an endpoint  for uploading images, so this is just a sample usage. Anyway, in the end the method should be POST and the response 202 is more appropriate.